conf/core/ConfigOptions.asciidoc

```diff
@@ -418,54 +459,13 @@ another changeset to the OSM API.
 
 The number of seconds after a successful write before submitting another changeset to the OSM API.
 
-=== changeset.buffer
-
-* Data Type: double
-* Default Value: `0.0`
-
-Value in degrees by which the specified AOI (convert.bounding.box) will be expanded when calculating
-changesets.  Setting this too large with some datasets may greatly increase the changeset
-derivation time.
-
 === changeset.description
 
 * Data Type: string
 * Default Value: `Hootenanny ingest`
 
 The text description that is written to the OSM API database with a changeset.
 
-=== changeset.reference.keep.entire.features.crossing.bounds
-
-* Data Type: bool
-* Default Value: `false`
-
-Overrides crop.keep.entire.features.crossing.bounds for the reference input when deriving
-changesets with a bounds.
-
-=== changeset.secondary.keep.entire.features.crossing.bounds
-
-* Data Type: bool
-* Default Value: `false`
-
-Overrides crop.keep.entire.features.crossing.bounds for the secondary input when deriving
-changesets with a bounds.
-
-=== changeset.reference.keep.only.features.inside.bounds
-
-* Data Type: bool
-* Default Value: `false`
-
-Overrides crop.keep.only.features.inside.bounds for the reference input when deriving
-changesets with a bounds.
-
-=== changeset.secondary.keep.only.features.inside.bounds
-
-* Data Type: bool
-* Default Value: `false`
-
-Overrides crop.keep.only.features.inside.bounds for the secondary input when deriving
-changesets with a bounds.
-
 === changeset.max.size
```

Comment:
These were being duplicated by the corresponding crop and convert bounding box params.